server/controllers/personSearch/ppcsSearchResultsController.ts

```diff
@@ -1,10 +1,16 @@
 import { NextFunction, Request, Response } from 'express'
 import { searchForPpcs } from '../../data/makeDecisionApiClient'
+import { isDefined, isEmptyStringOrWhitespace } from '../../utils/utils'
 
 async function get(req: Request, res: Response, next: NextFunction) {
   const { crn } = req.query
   const { user } = res.locals
 
+  if (!isDefined(crn) || isEmptyStringOrWhitespace(crn as string)) {
+    res.redirect(303, `no-ppcs-search-results?crn=${crn}`)
```

Comment:
Doesn't seem right to be passing CRN when it's been checked for being undefined/blank